LayoutTests/imported/w3c/web-platform-tests/import-maps/dynamic-integrity.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+let log;
```

Comment:
is there anything preventing the `let log` being defined on line 32? Then we can get rid of this script tag.